source/includes/fact-write-blocking-check.rst

```diff
@@ -0,0 +1,5 @@
+To see what state ``mongosync`` is in, call the :ref:`c2c-api-progress`
+API endpoint. The ``/progress`` end point returns a boolean value,
+``canWrite``. When ``canWrite`` is ``true``, it is safe to write to the
+destination cluster. If you write to the destination cluster while
```

Comment:
[consideration]

Using bullets to distinguish behavior between `canWrite` being true and false might help make this more scannable. For example:

```
- When ``canWrite`` is ``true``, it is safe to write to the destination cluster.
- When ``canWrite`` is ``false``, if you write to the destination cluster,
  ``mongosync`` exits.
```